app/usecase/usecase.go

```diff
@@ -2,47 +2,43 @@ package usecase
 
 import (
 	"fmt"
+	"github.com/short-d/kgs/app/usecase/event"
 	"time"
 
```

Comment:
Please remove the empty line and run `./scripts/format` to format the imports.

Reply:
Please break the changes to this file to a separate PR. We want to have logic small & ideally independent PRs which can be rolled back individually any time when bugs occur.